Track changes in desisurvey 0.9.0

dkirkby opened the pull request:
From the change log:
* Track API changes in desisurvey 0.9.0.
* The surveysim script is now called once per night, alternating with a
  surveyplan script that lives in the desisurvey package.
* See https://www.youtube.com/watch?v=vO1QZD_aCIo for a visualization of the
  full 5-year survey simulation that matches DESI-doc-1767-v3.

Unit tests are passing for me.  The tutorial reflects the latest changes.  A new shell script `bin/surveysim_baseline.sh` runs the full 5-year baseline survey.  This will be tagged as 0.8.0.

sbailey commented:
I confirm that I could follow the instructions in https://github.com/desihub/surveysim/blob/rebase/doc/tutorial.md to run a 5 year survey in ~5 hours.  It generated 1720 files but not `exposures.fits` which is the primary file that I previously used.  Has that been renamed or is there another way to convert a `progress.fits` file into the `exposures.fits` file now?

The top level context for these files is described in the tutorial doc, but eventually these should go into desidatamodel.  That will require determining canonical directory structure for operations and clearly distinguishing between what files mimic real operations vs. what are simulations-only.

Classes of files that I see:
* ephem_YEAR-MM-DD-YEAR-MM-DD.fits
* last_date.txt
* plan.fits
* plan_YEAR-MM-DD.fits (lots of these)
* plan_YEAR-MM-DD_bookmark.fits (lots of these, but not for every date)
* progress.fits (only one)
* progress_YEAR-MM-DD_bookmark.fits (lots of these)
* scheduler.fits
* stats.fits
* surveyinit.fits
* weather_{randseed}.fits

dkirkby commented:
I removed the automatic creation of `exposures.fits` since it was running every night, slowing things down, but only needed at the end of the survey.  Instead, I added instructions to the tutorial for creating it manually from the `progress.fits` that is updated each night:

https://github.com/desihub/surveysim/blob/rebase/doc/tutorial.md#simulate-initial-observing

dkirkby commented:
> The top level context for these files is described in the tutorial doc, but eventually these should go into desidatamodel. That will require determining canonical directory structure for operations and clearly distinguishing between what files mimic real operations vs. what are simulations-only.

I agree.  Do you want to create an issue for this (to keep me honest) or should I?
